manager/controllers/app/policy_compiler_interface.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
 // ConstructApplicationContext constructs ApplicationContext structure to send to Policy Compiler
+// TODO: Add DataCatalogID
```

Comment:
Do we expect policyManager to access the catalog searching for data, or we need to add some data context here? We have got it anyway from the catalog connector.

Reply:
@roee88 said that the policyManager will get the data from the catalog in the same way that it does it today when using egaria and OPA.